pkg/cache/tas_cache_test.go

```diff
@@ -1058,11 +1232,13 @@ func TestFindTopologyAssignment(t *testing.T) {
 					},
 				},
 			},
+			largestFit: true,
 		},
 	}
 	for name, tc := range cases {
 		t.Run(name, func(t *testing.T) {
 			ctx := context.Background()
+			features.SetFeatureGateDuringTest(t, features.LargestFitTAS, tc.largestFit)
 
```

Comment:
rather than update all test names, could have done:

```
if tc.largestFit {
  name = name + "; LargestFit"
} else [
 ... 
}
t.Run(name, func(t *testing.T) {
```

I'll leave this up to you if you want to do this way.

Reply:
Thanks! Done

Reply:
Oh, if the featureGate is the only change in the two scenarios I wanted to keep the same name of the test, which is impossible as it's a duplicated key in a map. Hence, I needed to add some difference, which was BestFit/LargestFit suffix. Alternatively I could change the test cases type from map to list to allow duplicated names, but I would prefer to keep this PR as simple as possible + changes in tests' names are easily revertible